cmd/config-seed/res/properties/go.mod

```diff
@@ -1 +1,3 @@
 module github.com/edgexfoundry/edgex-go/cmd/config-seed/res/properties
+
+go 1.12
```

Comment:
FYI PR https://github.com/edgexfoundry/edgex-go/pull/1681 has been created to remove this file altogether.